Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
sorry last comment is wrong, its posible use
```APPEND_FMT("%d", signed(regval));```
insted of ```int```?
in this cases are mutliples types of data, i think can be good avoid change it